### ISSUE TITLE
IntelliJ Plugin

### ISSUE THREAD

**Issue by NomNuggetNom** [chiquitinxx/grooscript-grails3-plugin#11]
An IntelliJ plugin would make this the best web developing experience I've ever had. What I really want is to have Groovy syntax hinting and such in the `grooscript` code blocks.

**Comment by chiquitinxx** [comment]
Yeah, good improvement :) No experience with Intellij idea plugins here.

**Comment by NomNuggetNom** [comment]
None here either :(. Maybe someone else could help...
